test/unit/RaisedButton.spec.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {assert} from 'chai';
+import RaisedButton from 'src/raised-button';
+import SvgIcon from 'src/svg-icon';
+
+describe('<RaisedButton />', () => {
+  it('if an SvgIcon is provided, renders the icon with the correct color', () => {
+    const path = <path d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z" />;
+    const icon = <SvgIcon color="red">{path}</SvgIcon>;
+    const wrapper = shallow(
+      <RaisedButton icon={icon} />
+    );
+
+    const svgIcon = wrapper.find('SvgIcon');
+    assert.strictEqual(svgIcon.length, 1, 'should have a SvgIcon');
+    assert.equal(svgIcon.node.props.color, 'red', 'should have color set as the prop');
```

Comment:
Also, this should be a unit test, not an integration test. Can you do it without using `<SvgIcon />` ?